src/utils/transactions/transactionInfoHelpers/tests/getScamFlag.test.ts

```diff
@@ -1,11 +1,11 @@
 import { getScamFlag } from '../getScamFlag';
 
 describe('scamDetect tests', () => {
-  const output = '[Message hidden due to suspicious content - ';
+  const output = 'Message hidden due to suspicious content - ';
   const strings: { [key: string]: string[] } = {
-    '👉 link.com': ['👉 link.com', ''],
+    '👉 link.com': ['👉 http://link.com', ''],
```

Comment:
tests should not change because scams do not necessarily put http inside their messages

Reply:
The http is for the output, not the input.